stockflux-securities-master/src/components/InputForm.js

```diff
@@ -85,10 +89,14 @@ const InputForm = () => {
             type="checkbox"
             onChange={() => setIsEnabled(!isEnabled)}
           />
-          <label className="toggle-switch" for="is-enabled-toggle" />
+          <label className="toggle-switch" htmlFor="is-enabled-toggle" />
         </div>
         <div className="input-submit-button-container">
-          <div className="input-submit-button">
+          <div
+            className={
```

Comment:
I think this might be down to prettier, but can we keep the same style of formatting used in the other apps?

    <div
      className={"input-submit-button" + (inProgress ? " in-progress" : "")}
    >

It could be worth using [classnames](https://www.npmjs.com/package/classnames) to simplify this. It seems to be used in the launcher and watchlist apps.

Reply:
Okay I'll bear this in mind for the future. It might be worth turning prettier off as it sometimes makes some wild choices!

Regarding classnames, Jack told me to use them when I started but Mark said he preferred we didn't.